File: src/main/java/com/matchFit/post/dto/response/GetMyPosts.java
package com.matchFit.post.dto.response;

import java.util.List;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class GetMyPosts {
    private final List<GetMyPost> posts;

    public static GetMyPosts from(List<GetMyPost> myPosts) {
        return new GetMyPosts(myPosts);
    }
}

File: src/main/kotlin/com/matchFit/post/dto/response/GetMyPosts.kt
package com.matchFit.post.dto.response

data class GetMyPosts(
    val posts: List<GetMyPost>
) {
    companion object {
        fun from(myPosts: List<GetMyPost>): GetMyPosts {
            return GetMyPosts(myPosts)
        }
    }
}
